README.rst

```diff
@@ -0,0 +1,86 @@
+xss-utils
+=============================
+
+|pypi-badge| |travis-badge| |codecov-badge| |doc-badge| |pyversions-badge|
+|license-badge|
+
+The ``README.rst`` file should start with a brief description of the repository,
+which sets it in the context of other repositories under the ``edx``
+organization. It should make clear where this fits in to the overall edX
+codebase.
+
+Utilities to prevent possible XSS attacks on Django/Mako templates.
+
+Overview (please modify)
```

Comment:
Drop "(please modify)".
Add some highlights of the code to be found in the repo.  Maybe links to READMEs for other directories.